src/ApiService/ApiService/onefuzzlib/Extension.cs

```diff
@@ -290,7 +290,7 @@ private sealed class Settings {
                 TypeHandlerVersion = "1.9",
                 AutoUpgradeMinorVersion = true,
                 Settings = new BinaryData(JsonSerializer.Serialize(new { commandToExecute = toExecuteCmd, fileUris = urlsUpdated }, _extensionSerializerOptions)),
-                ProtectedSettings = new BinaryData(managedIdentity)
+                ProtectedSettings = null
```

Comment:
You can probably remove this line entirely

Reply:
you can also remove the `managedIdentity` variable initialization as well

Reply:
we should also remove this from the linux config too

Reply:
Removed.